docs/docs/documentation/getting-started/installation/backend-config.md

```diff
@@ -33,6 +33,7 @@
 | POSTGRES_SERVER   | postgres | Postgres database server address |
 | POSTGRES_PORT     |   5432   | Postgres database port           |
 | POSTGRES_DB       |  mealie  | Postgres database name           |
+| POSTGRES_URL_OVERRIDE |   None   | Postgres URL override. Must be used with DB_ENGINE: postgres |
```

Comment:
It's not optional if using the variable. You must define that or it will not work because the provider is created based on DB_ENGINE.

Reply:
Yea, I understand the message you're trying to communicate, but I don't agree that the words achieve that. Otherwise that same message should be on all the `POSTGRES_x` env vars?

Something that talks to what it replaces would make more sense, e.g. (don't quote me, I've given this very little thought!!) "Postgres URL override. Used in place of POSTGRES_x/POSTGRES_y/... if you want to specify the full connection URI"

Reply:
This is what I came up with: "Optional Postgres URL override to use instead of POSTGRES_* variables. Must be used in conjunction with DB_ENGINE: postgres to have an effect."

Reply:
It's absolutely an improvement, I just still find this a weird addition `Must be used in conjunction with DB_ENGINE: postgres to have an effect.`, per previous comments.

Reply:
I'd be nervous to not add something similar as this is a quirk of the implementation.

The URL includes the dialect already, so its quite weird to have it specific to postgres if I am honest here.
If you don't specify it must still be used in conjunction with DB_ENGINE: postgres, someone may think that specifying DB_ENGINE is redundant, and remove it, only to find the instance has gone back to sqlite.

Reply:
Pretend this new variable doesn't exist..
If a user does not specify `DB_ENGINE: postgres`, they get a sqlite DB already, right?

I'm all for the docs being improved, but adding this exact message, to this _one_ environment variable, adds more confusion rather than simplifying things.